corehq/apps/translations/app_translations/upload_module.py

```diff
@@ -107,57 +107,61 @@ def update(self, rows):
         if self.no_items_text:
             self._update_translation(self.no_items_text, self.module.case_details.short.no_items_text)
 
-        self._update_case_search_labels(rows)
+        self._update_case_search_labels()
 
         return self.msgs
 
-    def _update_case_search_labels(self, rows):
+    def _update_case_search_labels(self):
         properties = self.module.search_config.properties
         displays = [row for row in self.condensed_rows if row['list_or_detail'] == 'case_search_display']
         hints = [row for row in self.condensed_rows if row['list_or_detail'] == 'case_search_hint']
-        if len(displays) != len(properties):
+
+        def partially_update_translations(incomplete_rows):
+            property_dict = {prop.name: prop for prop in properties}
+            for row in incomplete_rows:
+                prop_name = row.get('case_property')
+                if prop_name in property_dict:
+                    self._update_translation(row, property_dict[prop_name].hint)
```

Comment:
it wouldn't be `property_dict[prop_name].hint` for displays right?

Reply:
https://github.com/dimagi/commcare-hq/pull/34794/commits/04266bbf0ba5020e336be80517c8edfb5eb809fe